org.jboss.tools.maven.apt.feature/feature.properties

```diff
@@ -37,7 +37,7 @@ http://www.eclipse.org/legal/epl-v10.html\n\
 \n\
 Contributors:\n\
      Knowledge Computing Corp. - initial API and implementation\n
-     JBoss by Red Hat, Inc. - Packaging\n
+     Red Hat                   - Packaging\n
```

Comment:
This should probably be Red Hat, Inc.

Reply:
@nickboldt it looks identical to line 10, so why should it be different?

Reply:
@nickboldt I can put it back there easily if you think that's required. But there's no Inc. anywhere here for example:
https://github.com/jbosstools/jbosstools-central/pull/434/files